recipes/libmodbus/all/conanfile.py

```diff
@@ -42,74 +48,58 @@ def config_options(self):
     def configure(self):
         if self.options.shared:
             try:
-                del self.options.fPIC # once removed by config_options, need try..except for a second del
+                del self.options.fPIC
             except Exception:
                 pass
         try:
-            del self.settings.compiler.libcxx # for plain C projects only
+            del self.settings.compiler.libcxx
         except Exception:
             pass
         try:
-            del self.settings.compiler.cppstd # for plain C projects only
+            del self.settings.compiler.cppstd
```

Comment:
You can put this in the other try catch :)

Reply:
I don't think so.